test/models/role_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```

Comment:
what this file should describe. Should it be related to some model? If yes, e.g to User, rename it to `user_spec.rb` etc.